_includes/page-footer.html

```diff
@@ -1,9 +1,11 @@
 <footer class="site-footer">
-  <span class="site-footer-owner"><a href="{{ site.url }}">{{ site.title }}</a> is maintained by <a href="{{ site.author.url }}">{{ site.author.name }}</a>.</span>
-  <span class="site-footer-credits">This page was generated by <a
-href="https://pages.github.com">GitHub Pages</a>.
+  <span class="site-footer-owner">
+    <a href="{{ site.url }}">{{ site.title }}</a> von <a href="{{ site.author.url }}">{{ site.author.name }}</a>.<br />
+    Sketchnotes von <a href="https://twitter.com/Teapot4181">Lisa Maria Moritz</a>
+  </span>
+  <nav>
     <a href="/datenschutzhinweise.html">Datenschutzhinweise</a>
     <a href="/impressum.html">Impressum</a>
-  </span>
-
+  </nav>
+  <span class="site-footer-credits" lang="en">This page was generated by <a href="https://pages.github.com">GitHub Pages</a>.</span>
```

Comment:
I wasn't sure whether this actually adds value or whether it's merely a boilerplate remnant.

Reply:
We are using GitHub pages which provides their service for free. So I think it makes sense to mention them. 🙂